src/status_im2/contexts/onboarding/common/carousel/animation.cljs

```diff
@@ -95,18 +95,21 @@
 
 (defn drag-gesture
   [progress paused]
-  (->
-    (gesture/gesture-pan)
-    (gesture/enabled true)
-    (gesture/max-pointers 1)
-    (gesture/on-finalize
-     (fn [event]
-       (let [next?     (< (oops/oget event "translationX") (- drag-limit))
-             previous? (> (oops/oget event "translationX") drag-limit)]
-         (when next?
-           (update-progress progress paused (get-next-progress progress)))
-         (when previous?
-           (update-progress progress paused (get-previous-progress progress))))))))
+  (-> (gesture/gesture-pan)
+      (gesture/max-pointers 1)
+      (gesture/on-finalize
+       (fn [event]
+         (let [next?     (< (oops/oget event "translationX") (- drag-limit))
+               previous? (> (oops/oget event "translationX") drag-limit)]
+           (when next?
+             (update-progress progress paused (get-next-progress progress)))
+           (when previous?
+             (update-progress progress paused (get-previous-progress progress))))))))
```

Comment:
Refactored a little:

1. The code style for `->` 
2. removed `(gesture/enabled true)` since that's the default value and we are not passing any other value than true.